module/zfs/metaslab.c

```diff
@@ -639,7 +825,8 @@ void
 metaslab_class_evict_old(metaslab_class_t *mc, uint64_t txg)
 {
 	multilist_t *ml = &mc->mc_metaslab_txg_list;
-	hrtime_t now = gethrtime();
+	uint64_t now = gethrestime_sec();
+	uint_t delay = (metaslab_unload_delay_ms + 999) / 1000;
```

Comment:
It took me some time to understand this calculation.  Would you be ok with writing it this way?
```C
uint_t delay_sec = MAX(metaslab_unload_delay_ms / 1000, 1); /* min 1 second */
```

Reply:
I meant rounding it to the next second.  It is different and I think more reasonable in this case.

Reply:
OK.